schema_based_on_diagram.sql

```diff
@@ -0,0 +1,81 @@
+CREATE DATABASE hospital;
+
+CREATE TABLE medical_histories (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status VARCHAR,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE patients (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR,
+    date_of_birth DATE,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE invoices (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE invoice_items (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    unit_price DECIMAL,
+    quantity INT,
+    total_price DECIMAL,
+    invoice_id INT,
+    treatment_id INT,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    type VARCHAR,
+    name VARCHAR,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments_history (
```

Comment:
- [ ] Consider adding indexes for all foreign key
> for example `CREATE INDEX ON medical_histories (patient_id);`

Reply:
Thanks! Done 😄 :